assertk-jvm/src/main/kotlin/assertk/assertions/support/support.kt

```diff
@@ -0,0 +1,12 @@
+@file:JvmName("SupportJVMKt")
+package assertk.assertions.support
+
+actual
```

Comment:
same question here about modifier style - here it comes before the function annotations